client/overview/index.js

```diff
@@ -217,9 +219,23 @@ const OverviewPage = () => {
 					) }
 				</Notice>
 			) }
+			{ showResetAccountError && (
+				<Notice status="error" isDismissible={ false }>
+					{ __(
+						'There was a problem resetting your account. Please wait a few seconds and try again.',
+						'woocommerce-payments'
+					) }
+				</Notice>
+			) }
 			{ isDevMode ? (
 				<OverviewSandboxModeNotice
-					ctaAction={ () => setLivePaymentsModalVisible( true ) }
+					ctaAction={ () => {
+						recordEvent( 'wcpay_setup_live_payments_modal_open', {
```

Comment:
Small bit of housekeeping: new events should be added to the `client/tracks/event.d.ts` file. 

Because this is a JS file, you won't get a warning in the code, but if it was a TS file you would. 🙂

Reply:
Good call! Adding them all.